saleor/order/__init__.py

```diff
@@ -113,8 +113,8 @@ class OrderEvents:
     CHOICES = [
         (DRAFT_CREATED, "The draft order was created"),
         (DRAFT_CREATED_FROM_REPLACE, "The draft order with replace lines was created"),
-        (DRAFT_ADDED_PRODUCTS, "Some products were added to the draft order"),
-        (DRAFT_REMOVED_PRODUCTS, "Some products were removed from the draft order"),
+        (ADDED_PRODUCTS, "Some products were added to the order"),
```

Comment:
IMHO This will raise an error (correct me if I am wrong). 
You're dropping the enums, which are stored in DB. Saleor will try to map the db events to graphql's OrderEvent which will raise an error as we don't have such event types anymore.